src/Components/ServiceSelectionScene/ServiceSelectionScene.tsx

```diff
@@ -170,6 +288,14 @@ export class ServiceSelectionScene extends SceneObjectBase<ServiceSelectionScene
     }
   }
 
+  private getLogExpression(service: string, levelFilter: string) {
+    return `{${SERVICE_NAME_EXPR}=\`${service}\`}${levelFilter}`;
+  }
```

Comment:
The logs query must not use `__aggregated_metric__`. It's really just for metrics, not for logs.

Reply:
its so fast tho

Reply:
Fixed